es-app/src/SystemData.cpp

```diff
@@ -1656,7 +1656,7 @@ bool SystemData::isCheevosSupported()
 
 	const std::set<std::string> cheevosSystems = {
 		"arcade","atari2600","atari7800","atarilynx","colecovision","fbn","gamegear","gb",
-		"gba","gbc","genesis","mastersystem","megadrive","msx","msx2","n64","neogeo","nes","ngp",
+		"gba","gbc","genesis","mastersystem","megacd","megadrive","msx","msx2","n64","neogeo","nes","ngp",
```

Comment:
```suggestion
		"gba","gbc","genesis","intellivision","mastersystem","megacd","megadrive","msx","msx2","n64","neogeo","nes","ngp",
```

Please add "intellivision" as well - already tested it, works as well.